### DISCUSSION TITLE
When descriptions are added through WebUI, where are they stored?

### DISCUSSION THREAD

**Discussion by jessienab** [LycheeOrg/Lychee#2548] [Q&A] [answered]
> Set Description

> Enter a new description for this photo:

Where is the description stored? Within EXIF data in the image, or within the Lychee DB?

Thank you!

EDIT: based on a quick test, it appears it would be stored in the Lychee DB... but is there a way to have it written to the image? Such as in an alt-text way?

**Comment by ildyria** [comment, marked as answer]
> based on a quick test, it appears it would be stored in the Lychee DB.

That is correct.

> is there a way to have it written to the image? Such as in an alt-text way?

Not at the moment, and not planned on a short term. There are a lot more features I would like to add before that.
